src/components/Compare/CompareTableRow.tsx

```diff
@@ -92,7 +93,11 @@ const CompareTableRow = (props: {
           <FiberManualRecordIcon />
         </div>
         <div>
-          {locationName}
+          {locationName[0]}{' '}
+          {locationName[1] && <CountySuffix>{locationName[1]}</CountySuffix>}
```

Comment:
Consider handling the case of a county that doesn't have a space in its name (not sure if it exists, but maybe just in case?)

Reply:
that is a space between county name and its suffix (not in the specific county name itself)